components/CommonComponents/ShellBox/__tests__/__snapshots__/index.test.tsx.snap

```diff
@@ -0,0 +1,25 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
Jest has been setup. There is no test on the codebase yet. And I don't think we'll take another test package. Because Jest is very good for react (IMHO)

Reply:
> Is Jest already widely used in this repo? Is that conventional/necessary with Next.js? IIRC there was significant pushback against jest from some node core collaborators in other contexts, but I don't have much insight into this.

@tniessen I believe it's mostly because we've been using it on nodejs.dev. But I'm all in for finding a substitute if that's the will of the core collaborators. I'm all fine with Jasmine/Mocha. WDYT?

Reply:
I'd also be fine with native unit testing provided by Node.js, just not sure if it works well with DOM testing yet (haven't checked that yet)

Reply:
I don't have an opinion on this; I don't know what share of node core collaborators intend to work on this new repository structure at all, and I don't have an opinion on frontend testing frameworks myself. If the folks who do maintain the new website are happy with this, then that's great.

Reply:
I'd say let's keep at Jest for now then, and if somebody from Core opens an issue requesting a change, we can consider it :)